docs/image-sizes.md

```diff
@@ -0,0 +1,95 @@
+# Defining Image Sizes
+
+Altis provides some default image sizes, that you can use to render images in the theme.
+
+- Thumbnail
+- Medium
+- Large
+- Post Thumbnail
+- Full
+
+They have default dimensions, but you can override these either with code or in the Admin, under Settings > Media.
+
+![Media Settings](https://user-images.githubusercontent.com/30460/67079140-8b846000-f18a-11e9-8387-038b594f19aa.png)
+
+A lot of times, you'll find that you need additional image sizes for different contexts, depending on your theme's design.
+
+You can define additional image sizes with  the`add_image_size` function.
+
+This function accepts the following parameters:
+
+```php
+ * @param string     $name   Image size identifier.
+ * @param int        $width  Optional. Image width in pixels. Default 0.
+ * @param int        $height Optional. Image height in pixels. Default 0.
+ * @param bool|array $crop   Optional. Whether to crop images to specified width and height or resize.
+```
+The custom image sizes should be declared in the callback function to the `after_setup_theme` action.
+
+Example:
+```php
+add_action( 'after_setup_theme', __NAMESPACE__ . '\\theme_setup' );
+function theme_setup() {
+	// Set the image size by resizing the image proportionally (without distorting it):
+	add_image_size( 'custom-size', 220, 180 ); // 220 pixels wide by 180 pixels tall, soft proportional crop mode
+	
+	// Set the image size by cropping the image (not showing part of it):
+	add_image_size( 'custom-size', 220, 180, true ); // 220 pixels wide by 180 pixels tall, hard crop mode
+	
+	// Set the image size by cropping the image and defining a crop position:
+	add_image_size( 'custom-size', 220, 220, array( 'left', 'top' ) ); // Hard crop left top
+}
+```
+
+This will *not* add a UI for these named sizes under Settings > Media.
+
+### Cropping
+
+Cropping behavior for the image size is dependent on the value of $crop:
+1. If false (default), images will be scaled, not cropped.
+2. If an array in the form of array( x_crop_position, y_crop_position ):
+    - x_crop_position accepts 'left' 'center', or 'right'.
+    - y_crop_position accepts 'top', 'center', or 'bottom'.
+    Images will be cropped to the specified dimensions within the defined crop area.
+ 3. If true, images will be cropped to the specified dimensions using center positions.
+ 
+You can find more info in the [WordPress developer guide](https://developer.wordpress.org/themes/functionality/featured-images-post-thumbnails/#add-custom-featured-image-sizes)
+
+## Using The Custom Sizes
+
+Once you've defined custom image sizes, there are different ways you can use them.
+
+You can display a post thumbnail: `the_post_thumbnail( 'custom-size` );`
+
+You can display any uploaded image by its attachment ID: `wp_get_attachment_image_src( 1 , 'custom-size' );`
```

Comment:
I'd consider making this into a code example. Personal preference but I'm less keen on inline code examples. So something like:

```php
// Output the post thumbnail image tag:
the_post_thumbnail( 'custom-size' );
/// etc...
```